src/chat/components/InputToolbar.tsx

```diff
@@ -66,7 +101,7 @@ const InputToolbar: React.FC<IInputToolbar> = ({
       )}
       {hasCamera && (
         <PressableIcon
-          onPress={onPressSecondAction}
+          onPress={hasGallery ? openGallery : onPressSecondAction}
```

Comment:
What is secondAction?

Reply:
secondAction is default action, if not use, It will use for open Gallery. I removed hasGallery for easy to look

Reply:
you should change it to specific name because no one knows what the first/second action is and they have to go this file and read the code

Reply:
@baonguyenhsts we already discussed before, because we want more dynamic at this custom action. Do you have any suggestion or will I put more details on this action for can easy to use?